LpToJira/lp_to_jira.py

```diff
@@ -248,7 +248,12 @@ def main():
         version='devel',credential_store=credential_store)
 
     # Connect to the JIRA API
-    api = jira_api()
+    try:
+        api = jira_api()
+    except ValueError as exc:
+        print("Cannot initialize jira API, Abort.")
```

Comment:
```suggestion
        return "ERROR: Cannot initialize JIRA API."
```

This relies on `sys.exit()` optionally taking a string, and spitting that out on `stderr` with a non-zero return code - stderr being a better place than stdout for this kind of message, and fixing the case of JIRA.

Reply:
Thanks didn't know that, this is indeed better.